src/specialElHandlers.js

```diff
@@ -61,10 +64,14 @@ export default {
         if (!hasAttributeNS(toEl, null, 'multiple')) {
             var selectedIndex = -1;
             var i = 0;
-            var curChild = toEl.firstChild;
+            // We have to loop through children of fromEl, not toEl since nodes can be moved
+            // from toEl to fromEl directly when morphing.
+            // At the time this special handler is invoked, all children have already been morphed
+            // and appended to / removed from fromEl, so using fromEl here is safe and correct.
+            var curChild = fromEl.firstChild;
             while(curChild) {
                 var nodeName = curChild.nodeName;
```

Comment:
use `.toUpperCase()`

Reply:
Thanks, good catch. I added `.toUpperCase()` back in.